c_binding/libsg.c

```diff
@@ -331,6 +338,54 @@ bool _sg_is_vpd_page_supported(uint8_t *vpd_0_data, uint8_t page_code)
     return false;
 }
 
+int _sg_parse_vpd_80(char *err_msg, uint8_t *vpd_data, uint8_t *serial_num)
```

Comment:
Here you use `serial_num`, but caller use `vpd80`, can we be consistent for this?